File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        di = {}
        res = []
        for i in range(len(nums)):
            if target-nums[i] in di:
                return [di[target-nums[i]], i]
            else:
                di[nums[i]] = i
        return res
                

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    var res = []
    for(let i = 0 ; i < nums.length ; i++){
        for(let j = i + 1 ; j < nums.length ; j++){
            if(nums[i] + nums[j] == target){
                res.push(i);
                res.push(j);
            }
        }
    }
    return res;
};